### PR TITLE
Pretty links

### PR DESCRIPTION
**Pull request by Yago**
Offers multilines custom underline color instead of a single bottom line.

⚠️ We need to be careful of the `background` applied to the target types of link.

### CODE REVIEW

File: assets/components/atoms/link/link.scss [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
 
   .bg-dark & { color: $white; }
 
   &:hover,
   &.hover {
-    color: $link-hover-color;
     text-decoration: none;
-    border-bottom-color: $black !important;
-    .bg-dark & { border-bottom-color: $white !important; }
+    color: $link-color;
```
Comment (knuch) [inline assets/components/atoms/link/link.scss]: shouldn't this be red?

File: assets/components/atoms/link/link.scss [hunk 2]
```diff
@@ -1,27 +1,31 @@
 @charset 'utf-8';
 
-a:not([class*="btn"]) {
+a,
+a:hover,
+a:active { text-decoration: none; }
+
+a:not([class*="btn"]):not([class*="teaser"]):not([class*="card"]):not(.social-icon) {
```
Comment (knuch) [inline assets/components/atoms/link/link.scss]: this kind of breaks breadcrumbs. It needs some refactoring to use the new pretty link visual

Comment (Yago) [reply assets/components/atoms/link/link.scss]: Breadcrumb fixed with the footer branch (merged) 😉

File: assets/components/atoms/link/link.scss (hunk 1 above)
Comment (Yago) [reply assets/components/atoms/link/link.scss]: Not anymore according to the design updates...

Comment (knuch) [reply assets/components/atoms/link/link.scss]: @guillaume-berry Is that right?

**Pull request merged**